packages/cxl-ui/src/components/cxl-notification.js

```diff
@@ -0,0 +1,19 @@
+import { Notification } from '@vaadin/notification';
+
+export class CXLNotification extends Notification {
+  static get is() {
+    return 'cxl-notification';
+  }
+
+  renderer(root) {
+    if (root.firstElementChild) {
+      return;
+    }
+
+    [...this.childNodes].forEach((node) => {
+      root.appendChild(node);
+    });
+  }
```

Comment:
This could've used a a comment for "why is this necessary" @anoblet

Reply:
Makes sense. I will file a PR to add the comment.